tests/examples/pacer/nef/s3/cob.json

```diff
@@ -1,22 +1,22 @@
 {
   "appellate": false,
-  "contains_attachments": true,
```

Comment:
Also this seems like a different email example. Could we add  it a separate one?